packages/accessible-focus/package.json

```diff
@@ -6,9 +6,15 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic Inc.",
 	"sideEffects": false,
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
-	"calypso:src": "src/index.ts",
+	"type": "module",
+	"exports": {
```

Comment:
See more info about conditional exports: https://webpack.js.org/guides/package-exports/ and https://nodejs.org/api/packages.html#packages_package_entry_points

Reply:
I'm thinking this might technically be a breaking change, since external consumers of this package might be using a different build system, that doesn't know about conditional exports. What do you think?

Also, on that note, shouldn't we keep `main`? I'm guessing the new options override that?

Reply:
None of this is exceedingly important for this small package, mind you. But they're important questions to consider if we're planning on applying this more generally across the monorepo.

Reply:
I'd say this is a breaking change because we have effectively dropped support for old Node versions.
I didn't bump the version because this package is not published in NPM and only used in Calypso.

Regarding keeping `main`... It feels like a slippery slope. We are already saying this package only works in engines that support ESM imports. If we keep `main` just in case... why not `module`? Then what's the point of the migration? In that sense, I think we should also drop the CJS build.

I'm in the "Pure ESM" camp (from https://blog.sindresorhus.com/get-ready-for-esm-aa53530b3f77), but I don't feel strongly about it to be honest.

Reply:
> Also, on that note, shouldn't we keep `main`? I'm guessing the new options override that?

For most packages, especially those that are intended for browsers, both in Calypso and Gutenberg, I don't think that anyone ever used the CJS exports. I think this is a great opportunity to stop building and publishing them.

Reply:
Thank you both! After reading your comments, I agree that what we did here is the right way of going forward, with the aforementioned note that for published packages we should make sure we make this a breaking change. Dropping the CJS build in particular sounds like a great way of avoiding headaches on any project not making use of our build system.

Thank you for working on this experiment, @scinos!